SConscript-libffi

```diff
@@ -15,42 +15,48 @@ def find_tool(name):
     sys.exit(1)
 
 def subs(target, source, env):
-    with open(target[0].path, "w") as outf, open(source[0].path) as inf:
-        outf.write(inf.read()
-            .replace("@VERSION@", "3.2.1")
-            .replace("@TARGET@", "X86_WIN64")
-            .replace("@HAVE_LONG_DOUBLE@", "HAVE_LONG_DOUBLE")
-            .replace("@HAVE_LONG_DOUBLE_VARIANT@", "HAVE_LONG_DOUBLE_VARIANT")
-            .replace("@FFI_EXEC_TRAMPOLINE_TABLE@", "FFI_EXEC_TRAMPOLINE_TABLE")
-        )
+    with open(target[0].path, "w") as outf:
+        with open(source[0].path) as inf:
+		    outf.write(inf.read()
+                .replace("@VERSION@", "3.2.1")
+                .replace("@TARGET@", "X86_WIN64")
+                .replace("@HAVE_LONG_DOUBLE@", "HAVE_LONG_DOUBLE")
+                .replace("@HAVE_LONG_DOUBLE_VARIANT@", "HAVE_LONG_DOUBLE_VARIANT")
+                .replace("@FFI_EXEC_TRAMPOLINE_TABLE@", "FFI_EXEC_TRAMPOLINE_TABLE")
+            )
 
 def fix_target(target, source, env):
-    with open(target[0].path, "w") as outf, open(source[0].path) as inf:
-        for s in inf:
-            if s.startswith("#define FFI_TARGET_HAS_COMPLEX_TYPE"):
-                s = "//" + s
-            outf.write(s)
+    with open(target[0].path, "w") as outf:
+        with open(source[0].path) as inf:
+            for s in inf:
+                if s.startswith("#define FFI_TARGET_HAS_COMPLEX_TYPE"):
+                    s = "//" + s
+                outf.write(s)
 
 def sub_config(target, source, env):
-    with open(target[0].path, "w") as outf, open(source[0].path) as inf:
-        for s in inf:
-            if s.startswith("#undef"):
-                if s.split()[1] in [
-                    "HAVE_MEMCPY",
-                    "FFI_NO_RAW_API",
-                ]:
-                    s = "#define {} 1".format(s.split()[1])
-            outf.write(s)
+    with open(target[0].path, "w") as outf:
+        with open(source[0].path) as inf:
+            for s in inf:
+                if s.startswith("#undef"):
+                    if s.split()[1] in [
+                        "HAVE_MEMCPY",
+                        "FFI_NO_RAW_API",
+                    ]:
+                        s = "#define {} 1".format(s.split()[1])
+                outf.write(s)
 
 def remove_short(target, source, env):
-    with open(target[0].path, "w") as outf, open(source[0].path) as inf:
-        outf.write(inf.read().replace("SHORT", ""))
+    with open(target[0].path, "w") as outf:
+        with open(source[0].path) as inf:
+            outf.write(inf.read().replace("SHORT", ""))
 
 Import("env")
 
 if not os.path.exists("external/libffi-3.2.1/configure"):
     tarfile.open("external/libffi-3.2.1.tar.gz").extractall("external")
+
 if sys.platform == "win32":
+    env.Command("external/libffi-3.2.1/include/ffi.h.in", "external/libffi-3.2.1.tar.gz", lambda target, source, env: tarfile.open(source[0].path).extractall("external"))
```

Comment:
This shouldn't be here. See https://github.com/ghewgill/neon-lang/commit/22e1149055143720e1feaba89b34a36007795448 for the commit that introduces this.